sse2neon.h

```diff
@@ -8995,14 +8995,17 @@ FORCE_INLINE int _sse2neon_ctzll(unsigned long long x)
 #define SSE2NEON_CMPSTR_SET_UPPER(var, imm) \
     const int var = (imm & 0x01) ? 8 : 16
 
-#define SSE2NEON_CMPESTRX_COMP_AGG(la, lb, imm8)                   \
+#define SSE2NEON_CMPESTRX_GET_LEN(a, b, la, lb) \
+    int tmp1 = la ^ (la >> 31);                 \
+    la = tmp1 - (la >> 31);                     \
+    int tmp2 = lb ^ (lb >> 31);                 \
+    lb = tmp2 - (lb >> 31);                     \
+    la = SSE2NEON_MIN(la, bound);               \
+    lb = SSE2NEON_MIN(lb, bound)
+
+#define SSE2NEON_COMP_AGG(la, lb, imm8, CMPESTRX)                  \
```

Comment:
You shall add some comment to illustrate the functionality and motivations of these macros, otherwise it is not straightforward to maintain.